src/Domain/Collector.php

```diff
@@ -938,6 +938,8 @@ private function getMaximum(array $array)
      */
     private function divide(float $x, float $y): float
     {
+        if($x==$y==0)
```

Comment:
@svikramjeet When exactly this be `0`? This really happens?

Reply:
I don't really see why this is necessary. The method already takes care of `y` being 0. If this is necessary it does not seem like something that should be handled in the `divide` method, at least for me it does not make sense that it returns 100 if they both are 0.